x-pack/plugins/upgrade_assistant/__jest__/client_integration/overview/fix_issues_step/fix_issues_step.test.tsx

```diff
@@ -8,22 +8,24 @@
 import { act } from 'react-dom/test-utils';
 import { deprecationsServiceMock } from 'src/core/public/mocks';
 
-import * as mockedResponses from './mocked_responses';
-import { setupEnvironment } from '../../helpers';
+import { setupEnvironment, kibanaDeprecationsServiceHelpers } from '../../helpers';
 import { OverviewTestBed, setupOverviewPage } from '../overview.helpers';
+import { esDeprecations, esDeprecationsEmpty } from './mocked_responses';
 
 describe('Overview - Fix deprecation issues step', () => {
   let testBed: OverviewTestBed;
   const { server, httpRequestsMockHelpers } = setupEnvironment();
+  const {
+    criticalDeprecations: mockedCriticalKibanaDeprecations,
+    warningDeprecations: mockedWarningKibanaDeprecations,
```

Comment:
Nit: If we export these vars with the `mocked` prefix we won't need to rename them in the test files.